userspace/libsinsp/threadinfo.cpp

```diff
@@ -1537,9 +1529,10 @@ std::unique_ptr<sinsp_threadinfo> sinsp_thread_manager::new_threadinfo() const
  */
 bool sinsp_thread_manager::add_thread(sinsp_threadinfo *threadinfo, bool from_scap_proctable)
 {
-#ifdef GATHER_INTERNAL_STATS
-	m_added_threads->increment();
-#endif
+	if (m_inspector->m_sinsp_stats_v2)
+	{
+		m_inspector->m_sinsp_stats_v2->m_n_added_threads++;
+	}
```

Comment:
I would move this to the end of the function because here it is not counting the `added_threads`, the table could be full. This counts the `requests` of adding a thread